packages/create-vite/template-vue-ts/src/style.css

```diff
@@ -0,0 +1,65 @@
+:root {
```

Comment:
It might be better to be `index.css`?

Reply:
I think that for Vue style.css is more idiomatic